db/setup/schema.sql

```diff
@@ -2,10 +2,10 @@ CREATE DATABASE ft_transcendence;
 
 \c ft_transcendence;
 
-CREATE TABLE IF NOT EXISTS users(
-    ID          UUID DEFAULT gen_random_uuid()  PRIMARY KEY,
-    USERNAME    VARCHAR(20)                     NOT NULL UNIQUE,
-    EMAIL       VARCHAR(50)                     NOT NULL UNIQUE,
-    AVATAR      VARCHAR(150),
-    CREATED_AT  TIMESTAMPTZ                     NOT NULL DEFAULT now()
+CREATE TABLE IF NOT EXISTS Users(
+  "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
```

Comment:
I was concerned about getting collisions of uuids when we don't check that we already have one like the one we just created, but [this](https://stackoverflow.com/questions/18708496/how-to-be-sure-uuid-is-not-duplicated) convinced me that this is not likely at all.